### PR TITLE
HARP-5590 Fix code snippets references in examples

### PR DESCRIPTION
**Pull request by musculman**
* Adapt regex of modules to remove the scope from TypeDoc generation
* Render each module's README for TypeDoc

Signed-off-by: Ignacio Julve <41909512+musculman@users.noreply.github.com>

### CODE REVIEW

File: @here/harp-examples/src/camera_free.ts [hunk 1]
```diff
@@ -33,8 +33,12 @@ import "three/examples/js/controls/TransformControls";
  * The app enables to change the position of the camera: translate it, rotate it
  * as well as change the point of view (to the one the user would actually see).
  *
+ * ```typescript
+ * [[include:harp_gl_freecamera_app_0.ts]]
+ * ```
+ *
  */
-export namespace FreeCameraApp_DebugingToolExample {
+export namespace FreeCameraAppDebugingToolExample {
```
Comment (nzjony) [inline @here/harp-examples/src/camera_free.ts]: Debuging -> Debugging

Comment (musculman) [reply @here/harp-examples/src/camera_free.ts]: Done!

File: @here/harp-examples/src/textcanvas_dynamic.ts [hunk 2]
```diff
@@ -25,8 +25,6 @@ import {
  * This example showcases how [[TextCanvas]] can handle dynamic loading of multiple [[FontCatalog]]
  * assets, as well as dynamic real-time text styling.
  *
- * For more information regarding basic [[TextCanvas]] initialization and usage, please check:
```
Comment (nzjony) [inline @here/harp-examples/src/textcanvas_dynamic.ts]: why was this removed? TextCanvasMinimalExample still seems to exist.

File: @here/harp-examples/src/datasource_webtile.ts [hunk 3]
```diff
@@ -10,6 +10,23 @@ import { CopyrightElementHandler, MapView } from "@here/harp-mapview";
 import { WebTileDataSource } from "@here/harp-webtile-datasource";
 import { appCode, appId } from "../config";
 
+// tslint:disable:max-line-length
+/**
+ * A simple example using the webtile data source. Tiles are retrieved from
+ * ```typescript
```
[the remark this answers is not in the document]
Comment (musculman) [reply @here/harp-examples/src/datasource_webtile.ts]: Not really, I will remove it.

File: @here/harp-examples/src/textcanvas_dynamic.ts (hunk 2 above)
Comment (musculman) [reply @here/harp-examples/src/textcanvas_dynamic.ts]: It is not currently available in harp.gl, but will be added back in the future, so I remove it until then
